licenses/print.py

```diff
@@ -0,0 +1,143 @@
+from .models import LicenseRequest
```

Comment:
As the only function in this module indicates, it is about generation of a file and not about printing it. This should be reflected in the name, e.g. not `print.py`.